etc/pot/pot.conf.sample

```diff
@@ -10,6 +10,9 @@
 # This is the cache used to import/export pots
 # POT_CACHE=/var/cache/pot
 
+# This is where pot is going to store temporary files
+# POT_TMP=/tmp
```

Comment:
Another question is, if having a default of /tmp/.pot wouldn't make more sense (to keep those temporary files tucked away nicely). Right now a lot of action in /tmp.

Reply:
I'm using as default the value used before for consistency, so all the files are still written in the same folder, after an update

Reply:
Ok :)